lib/adapters/http.js

```diff
@@ -85,6 +85,10 @@ module.exports = function httpAdapter(config) {
       method: config.method.toUpperCase(),
       headers: headers,
       agent: agent,
+      agents: {
```

Comment:
@JitixPhotobox i see in the `follow-redirects` documentation that the object should be:
```
agents: {
  https: Agent,
  http: Agent
}
```
Or am I missing something?